provision/testdata/.acirc

```diff
@@ -1,3 +1,2 @@
-export AZ_VIRTUAL_NETWORK="ul_akstest"
-export AZ_CAPIC_SUBNET_ID="/subscriptions/ed1aefee-3242-413c-899e-fb8bfe0fdee2/resourceGroups/CAPIC_aks1_ul_akstest_westus/providers/Microsoft.Network/virtualNetworks/ul_akstest/subnets/subnet-43.0.1.0_24"
-export AZ_CAPIC_REGION="westus"
\ No newline at end of file
+|public subnets|: ['subnet-06cdb0b67bf86fcbf', 'subnet-0814cdfa1911664e7']
+|private subnets|: ['subnet-093bf9479f50168f7', 'subnet-00a54d51ba268fe29']
```

Comment:
is this changed intended?

Reply:
it is not intended, looks like it is generated content